src/Mammon/Dockerfile

```diff
@@ -0,0 +1,25 @@
+#See https://aka.ms/customizecontainer to learn how to customize your debug container and how Visual Studio uses this Dockerfile to build your images for faster debugging.
+
+FROM mcr.microsoft.com/dotnet/aspnet:8.0 AS base
+USER app
+WORKDIR /app
+EXPOSE 8080
+EXPOSE 8081
+
+FROM mcr.microsoft.com/dotnet/sdk:8.0 AS build
```

Comment:
what I've been bothering Art with is that we shouldn't be building nor publishing in the docker file, but on the pipeline and then copy the binaries from there.

Reply:
we talked, let's ignore